test/integration/default/serverspec/pgd_spec.rb

```diff
@@ -29,14 +13,14 @@
   it { should be_running }
 end
 
-describe file('/opt/pgd') do
-  it { should be_directory }
-end
-
-describe file('/opt/vrt') do
-  it { should be_directory }
+%w(/opt/pgd/settings.py /opt/pgd/selected_proteins.txt).each do |p|
```

Comment:
https://docs.djangoproject.com/en/1.4/releases/1.4/#updated-default-project-layout-and-manage-py

``__init__.py``, ``settings.py``, ``urls.py``, ``views.py``, and ``wsgi.py`` are now in ``./pgd`` as seen from the top level.  ``manage.py`` is still in the top level, though.

I'm getting a failure from ``tkc test`` at "Running syncdb for PGD" and I think it's related to this issue -- that, or the repo being cloned into ``/opt/pgd/source``, it's hard to say.